### PR TITLE
Feature/lyric extrac features

### PR DESCRIPTION
**Pull request by Johny-Code**

### CODE REVIEW

File: tools/extract_features_from_lyric.py [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+        pos_tags_count = get_pos_tags_count(lines, nlp)
+        sentiment_polarity, sentiment_subjectivity = get_sentiment(lyric)
+
+        row = [lyrics_vector, echoisms, duplicate_lines, title_in_lyric, verb_present_freq, verb_past_freq, verb_future_freq,
+                pos_tags_count['ADJ'], pos_tags_count['PUNCT'], sentiment_polarity, sentiment_subjectivity]
+
+        rows.append(row)
+        ids.append(index)
+
+        #In the end, we obtained the best results just by using the following features:
```
Comment (michsak) [inline tools/extract_features_from_lyric.py]: This type of redundant code (long comments) should be avoided where it's possible - e.g., you have variable names like that, why do u include that in the comment as well? If it comes to echoism, lyrics vector, sentiment, subjectivity degree you can put it near the variable initialization i.e. for example, above echoism=get_echoism(...,...) line

File: tools/extract_features_from_lyric.py [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+        ## Past tense verbs,
+        ## Present tense verbs, 
+        ## Future tense verbs, 
+        ## ADJ, -> Adjective
+        ## PUNCT, -> Punctuation
+        ## Sentiment -> TextBlob returns polarity and subjectivity of a sentence. Polarity lies between [-1,1], -1 defines a negative sentiment and 1 defines a positive sentiment.
+        ## Subjectivity degree -> Subjectivity quantifies the amount of personal opinion and factual information contained in the text. lies between [0,1]
+    
+    features_df = pd.DataFrame(rows, columns=['lyrics_vector', 'echoisms', 'duplicate_lines', 'title_in_lyric', 'verb_present_freq', 'verb_past_freq', 'verb_future_freq',
+                                              'count_ADJ', 'count_PUNCT', 'sentiment_polarity', 'sentiment_subjectivity'], index=ids)
```
Comment (michsak) [inline tools/extract_features_from_lyric.py]: formatting

**Review by michsak — CHANGES_REQUESTED** [review]

File: tools/extract_features_from_lyric.py (hunk 1 above)
Comment (Johny-Code) [reply tools/extract_features_from_lyric.py]: Right!

File: tools/extract_features_from_lyric.py (hunk 2 above)
Comment (Johny-Code) [reply tools/extract_features_from_lyric.py]: Right!

**Pull request merged**